subcommand/health-sync/dataplane_monitor.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
This file is also a copy paste of the existing implementation removed in https://github.com/hashicorp/consul-ecs/pull/209